docs/site/Implementing-auto-migration.md

```diff
@@ -0,0 +1,158 @@
+---
+lang: en
+title: 'Implementing Auto-Migration'
```

Comment:
I am not sure if the readers new to LoopBack will understand what "auto-migration" means. How about the following alternative titles?

- Database migrations
- How to update your database schema
- Auto-updating database schemas

Reply:
+1 on changes in the title. I think originally it was coming from the LB3 corresponding page: https://loopback.io/doc/en/lb3/Implementing-auto-migration.html